### PR TITLE
Update example for Variants with many values

### PR DESCRIPTION
**Pull request by vinamelody**
Updating the example snippet based on
https://github.com/ben-rogerson/twin.macro/discussions/425

### CODE REVIEW

File: docs/styled-component-guide.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ const Component = ({ hasBg }: ContainerProps) => (
 
 ## Variants with many values
 
-When a variant has many values (eg: `variant="light/dark/etc"`), name the class set in an object and use a prop to grab the entry containing the styles:
+When a variant has many values (eg: `variant="light/dark/etc"`), name the class set in an object and use a prop to grab the entry containing the styles. Note that you must return a function as follow:
```
Comment (ben-rogerson) [inline docs/styled-component-guide.md:74]: Grammar fix:

```suggestion
When a variant has many values (eg: `variant="light/dark/etc"`), name the class set in an object and use a prop to grab the entry containing the styles. Note that you must return a function as follows:
```

**Review by ben-rogerson — CHANGES_REQUESTED** [review]
Just a small grammar fix, looks great otherwise - thanks :)

File: docs/styled-component-guide.md (hunk 1 above)
Comment (vinamelody) [reply docs/styled-component-guide.md]: Sure, thanks for the review 👍

**Comment by ben-rogerson** [comment]
thanks again 👋

**Comment by vinamelody** [comment]
^^ I'm the one who needs to thank you!

**Pull request merged**